documentation/docs/20-core-concepts/30-form-actions.md

```diff
@@ -178,8 +178,17 @@ export const actions = {
 
 ```diff
 /// file: src/routes/login/+page.svelte
+<script>
+	import { page } from '$app/stores'
+
+	export let form
+</script>
+
 <form method="POST" action="?/login">
 -	<input name="email" type="email">
++	{#if form?.missing || form?.incorrect}
++		<p>{$page.status} There are errors in your form submission. See below</p>
++	{/if}
```

Comment:
do these three lines add much? The goal is to keep these samples as direct and concise as possible

```suggestion
```